src/de.f90

```diff
@@ -412,10 +415,11 @@ function diver(func, &
     endif
 
     !Prepare to return the output parameters and derived quantities at the minimum
-    if (present(paramsPlus)) then
-      paramsPlus(1:run_params%D) = params
-      paramsPlus(run_params%D+1:run_params%D+run_params%D_derived) = &
-       BF%vectors_and_derived(1,run_params%D+1:run_params%D+run_params%D_derived)
+    if (present(bestFitParams)) then
+      bestFitParams(1:run_params%D) = params
```

Comment:
now that the array has been split apart, this can just be `bestFitParams = params`, no?

Reply:
true - done